invenio_accounts/sessions.py

```diff
@@ -56,10 +54,10 @@ def add_session(session=None):
     user_id, sid_s = session['user_id'], session.sid_s
     session_activity = SessionActivity(user_id=user_id, sid_s=sid_s)
     try:
-        _datastore.db.session.add(session_activity)
-        _datastore.commit()
+        _db.session.add(session_activity)
```

Comment:
Wait, we'd still need the try/except block otherwise an exception will just cause the entire app to crash, no?